### PR TITLE
feat: improve audio file support (m4a, AAC, FLAC, OGG)

### PR DESCRIPTION
**Pull request by spaenleh**
In this PR:
- added new audio formats
- used local files instead of remote ones for storybook tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
     "@emotion/cache": "~11.13.1",
     "@emotion/react": "11.13.3",
     "@emotion/styled": "11.13.0",
-    "@graasp/sdk": "4.26.0",
+    "@graasp/sdk": "github:graasp/graasp-sdk#642-m4a-support",
```
Comment (ReidyT) [inline package.json:119]: reminder ^^

File: src/items/FileAudio.stories.tsx [hunk 2]
```diff
@@ -17,22 +17,53 @@ export default {
...
+    url: '/test-assets/sample.wav',
     type: MimeTypes.Audio.WAV, // should be wav format
   },
-};
+} satisfies Story;
+
+export const M4AAudio = {
+  args: {
+    id: 'some-audio-file-id',
+    // todo: use local file
```
Comment (ReidyT) [inline src/items/FileAudio.stories.tsx:44]:
```suggestion

```

File: src/items/FileAudio.stories.tsx [hunk 3]
```diff
@@ -17,22 +17,53 @@ export default {
...
+    url: '/test-assets/sample.m4a',
+    type: MimeTypes.Audio.MP4_Apple,
+  },
+  name: 'M4A Audio',
+} satisfies Story;
+
+export const AACAudio = {
+  args: {
+    id: 'some-audio-file-id',
+    // todo: use local file
```
Comment (ReidyT) [inline src/items/FileAudio.stories.tsx:54]:
```suggestion

```

File: src/items/FileAudio.stories.tsx [hunk 4]
```diff
@@ -17,22 +17,53 @@ export default {
...
+    url: '/test-assets/sample.aac',
+    type: MimeTypes.Audio.AAC, // should be wav format
+  },
+  name: 'AAC Audio',
+} satisfies Story;
+
+export const OGGAudio = {
+  args: {
+    id: 'some-audio-file-id',
+    // todo: use local file
```
Comment (ReidyT) [inline src/items/FileAudio.stories.tsx:64]:
```suggestion

```

File: src/items/FileAudio.stories.tsx [hunk 5]
```diff
@@ -17,22 +17,53 @@ export default {
...
+    url: '/test-assets/sample.ogg',
+    type: MimeTypes.Audio.OGG, // should be wav format
```
Comment (ReidyT) [inline src/items/FileAudio.stories.tsx:66]:
```suggestion
    type: MimeTypes.Audio.OGG, 
```

File: src/items/FileAudio.stories.tsx [hunk 6]
```diff
@@ -17,22 +17,53 @@ export default {
...
+  },
+  name: 'M4A Audio',
+} satisfies Story;
+
+export const AACAudio = {
+  args: {
+    id: 'some-audio-file-id',
+    // todo: use local file
+    url: '/test-assets/sample.aac',
+    type: MimeTypes.Audio.AAC, // should be wav format
```
Comment (ReidyT) [inline src/items/FileAudio.stories.tsx:56]: It is necessary?
```suggestion
    type: MimeTypes.Audio.AAC, 
```

**Review by ReidyT — APPROVED** [review]
LGTM 🚀

**Review by pyphilia — APPROVED** [review]
LGTM

**Pull request merged**